libs/common-docs/src/lib/common/schematics/schematics.component.html

```diff
@@ -0,0 +1,129 @@
+<demo-section>
+  <div id="content" class="content-box">
+    <a href="https://github.com/valor-software/ngx-bootstrap">
+      <div class="image-responsive img-fluid main-logo">
+        <img class="mx-auto center-block d-block" src="assets/images/logos/ngx-bootstrap-logo.svg"
+             alt="ngx-bootstrap" width="200" height="200">
+      </div>
+      <h1 align="center">ngx-bootstrap</h1>
+    </a>
+
+    <p align="center">
+      The best way to quickly integrate <a href="https://getbootstrap.com/docs/3.3/">Bootstrap 3</a> or <a
+      href="https://getbootstrap.com/docs/4.0">Bootstrap 4</a> Components with <a href="https://angular.io/">Angular</a>
+    </p>
+
+    <section class="section">
+      <h2 id="getting-started">
+        Schematics
+      </h2>
+    </section>
+
+    <section class="section">
+      <p>
+        Library ngx-bootstrap contains Angular Cli Schematics to allow easier installation. After installing npm packages it will be available through the Angular CLI. You can update you project with ngx-bootstrap library using this command.
+      </p>
+      <pre class="prettyprint lang-bash prettyprinted"><span class="pln">ng add ngx-bootstrap</span></pre>
+      <p>To learn more regarding installation our library you can <a routerLink="/documentation" fragment="installation">there</a></p>
+      <p>In case you install ngx-bootstrap with Angular Cli Schematics you will get such updates:</p>
+      <ul id="installation">
+        <li><p><code>package.json</code></p>
+          <p>List of dependencies will be updated with 2 new packages <code>"bootstrap": "^4.5.0"</code>, <code>"ngx-bootstrap": "7.0.0-rc.1"</code>. After packages will be installed and node_modules folder will be updated also.</p>
+        </li>
+        <li id="style_updates"><p>Depends on which style extension is used in project, schematics will add necessary imports or styles links. It allows use SCSS or CSS extensions.</p>
+          <p>In case CSS extension - <code>angular.json</code> file wil be updated.</p>
+          <pre class="prettyprint lang-js prettyprinted">
+<span class="kwd">"styles"</span><span class="pun">: [</span>
+    "<span class="lit">./node_modules/bootstrap/dist/css/bootstrap.min.css</span>",
+    "<span class="lit">./node_modules/ngx-bootstrap/datepicker/bs-datepicker.css</span>",
+    <span class="pln">  &hellip;</span>
```

Comment:
<span class="pln">  &hellip;</span> causes some weired ... symbols in the `code` block